<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IO16</class>
 <widget class="QWidget" name="IO16">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Debounce Period</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="debounce_edit">
         <property name="text">
          <string>100</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="debounce_save">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Port</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="port_box">
         <item>
          <property name="text">
           <string>A</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>B</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Pin</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="pin_box">
         <item>
          <property name="text">
           <string>0</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>7</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Direction</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="direction_box">
         <item>
          <property name="text">
           <string>Input</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Output</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Value</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="value_box"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="save_button">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Port A:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <property name="horizontalSpacing">
        <number>20</number>
       </property>
       <property name="verticalSpacing">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item row="1" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Value</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Config</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Direction</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="av0">
         <property name="text">
          <string>High</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="av1">
         <property name="text">
          <string>High</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="av2">
         <property name="text">
          <string>High</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QLabel" name="av3">
         <property name="text">
          <string>High</string>
         </property>
        </widget>
       </item>
       <item row="1" column="5">
        <widget class="QLabel" name="av4">
         <property name="text">
          <string>High</string>
         </property>
        </widget>
       </item>
       <item row="1" column="6">
        <widget class="QLabel" name="av5">
         <property name="text">
          <string>High</string>
         </property>
        </widget>
       </item>
       <item row="1" column="7">
        <widget class="QLabel" name="av6">
         <property name="text">
          <string>High</string>
         </property>
        </widget>
       </item>
       <item row="1" column="8">
        <widget class="QLabel" name="av7">
         <property name="text">
          <string>High</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="ad0">
         <property name="text">
          <string>Input</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="ad1">
         <property name="text">
          <string>Input</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QLabel" name="ad2">
         <property name="text">
          <string>Input</string>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QLabel" name="ad3">
         <property name="text">
          <string>Input</string>
         </property>
        </widget>
       </item>
       <item row="2" column="5">
        <widget class="QLabel" name="ad4">
         <property name="text">
          <string>Input</string>
         </property>
        </widget>
       </item>
       <item row="2" column="6">
        <widget class="QLabel" name="ad5">
         <property name="text">
          <string>Input</string>
         </property>
        </widget>
       </item>
       <item row="2" column="7">
        <widget class="QLabel" name="ad6">
         <property name="text">
          <string>Input</string>
         </property>
        </widget>
       </item>
       <item row="2" column="8">
        <widget class="QLabel" name="ad7">
         <property name="text">
          <string>Input</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="ac0">
         <property name="text">
          <string>Pull Up</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="ac1">
         <property name="text">
          <string>Pull Up</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QLabel" name="ac2">
         <property name="text">
          <string>Pull Up</string>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="QLabel" name="ac3">
         <property name="text">
          <string>Pull Up</string>
         </property>
        </widget>
       </item>
       <item row="3" column="5">
        <widget class="QLabel" name="ac4">
         <property name="text">
          <string>Pull Up</string>
         </property>
        </widget>
       </item>
       <item row="3" column="6">
        <widget class="QLabel" name="ac5">
         <property name="text">
          <string>Pull Up</string>
         </property>
        </widget>
       </item>
       <item row="3" column="7">
        <widget class="QLabel" name="ac6">
         <property name="text">
          <string>Pull Up</string>
         </property>
        </widget>
       </item>
       <item row="3" column="8">
        <widget class="QLabel" name="ac7">
         <property name="text">
          <string>Pull Up</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_53">
         <property name="text">
          <string>Pin</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_55">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_56">
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label_57">
         <property name="text">
          <string>2</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLabel" name="label_58">
         <property name="text">
          <string>3</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QLabel" name="label_59">
         <property name="text">
          <string>4</string>
         </property>
        </widget>
       </item>
       <item row="0" column="6">
        <widget class="QLabel" name="label_60">
         <property name="text">
          <string>5</string>
         </property>
        </widget>
       </item>
       <item row="0" column="7">
        <widget class="QLabel" name="label_61">
         <property name="text">
          <string>6</string>
         </property>
        </widget>
       </item>
       <item row="0" column="8">
        <widget class="QLabel" name="label_62">
         <property name="text">
          <string>7</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Port B:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="horizontalSpacing">
        <number>20</number>
       </property>
       <item row="1" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Value</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Config</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Direction</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="bv0">
         <property name="text">
          <string>High</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="bv1">
         <property name="text">
          <string>High</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="bv2">
         <property name="text">
          <string>High</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QLabel" name="bv3">
         <property name="text">
          <string>High</string>
         </property>
        </widget>
       </item>
       <item row="1" column="5">
        <widget class="QLabel" name="bv4">
         <property name="text">
          <string>High</string>
         </property>
        </widget>
       </item>
       <item row="1" column="6">
        <widget class="QLabel" name="bv5">
         <property name="text">
          <string>High</string>
         </property>
        </widget>
       </item>
       <item row="1" column="7">
        <widget class="QLabel" name="bv6">
         <property name="text">
          <string>High</string>
         </property>
        </widget>
       </item>
       <item row="1" column="8">
        <widget class="QLabel" name="bv7">
         <property name="text">
          <string>High</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="bd0">
         <property name="text">
          <string>Input</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="bd1">
         <property name="text">
          <string>Input</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QLabel" name="bd2">
         <property name="text">
          <string>Input</string>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="QLabel" name="bd3">
         <property name="text">
          <string>Input</string>
         </property>
        </widget>
       </item>
       <item row="3" column="5">
        <widget class="QLabel" name="bd4">
         <property name="text">
          <string>Input</string>
         </property>
        </widget>
       </item>
       <item row="3" column="6">
        <widget class="QLabel" name="bd5">
         <property name="text">
          <string>Input</string>
         </property>
        </widget>
       </item>
       <item row="3" column="7">
        <widget class="QLabel" name="bd6">
         <property name="text">
          <string>Input</string>
         </property>
        </widget>
       </item>
       <item row="3" column="8">
        <widget class="QLabel" name="bd7">
         <property name="text">
          <string>Input</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="bc0">
         <property name="text">
          <string>Pull Up</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QLabel" name="bc1">
         <property name="text">
          <string>Pull Up</string>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QLabel" name="bc2">
         <property name="text">
          <string>Pull Up</string>
         </property>
        </widget>
       </item>
       <item row="4" column="4">
        <widget class="QLabel" name="bc3">
         <property name="text">
          <string>Pull Up</string>
         </property>
        </widget>
       </item>
       <item row="4" column="5">
        <widget class="QLabel" name="bc4">
         <property name="text">
          <string>Pull Up</string>
         </property>
        </widget>
       </item>
       <item row="4" column="6">
        <widget class="QLabel" name="bc5">
         <property name="text">
          <string>Pull Up</string>
         </property>
        </widget>
       </item>
       <item row="4" column="7">
        <widget class="QLabel" name="bc6">
         <property name="text">
          <string>Pull Up</string>
         </property>
        </widget>
       </item>
       <item row="4" column="8">
        <widget class="QLabel" name="bc7">
         <property name="text">
          <string>Pull Up</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_54">
         <property name="text">
          <string>Pin</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_63">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_64">
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label_65">
         <property name="text">
          <string>2</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLabel" name="label_66">
         <property name="text">
          <string>3</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QLabel" name="label_67">
         <property name="text">
          <string>4</string>
         </property>
        </widget>
       </item>
       <item row="0" column="6">
        <widget class="QLabel" name="label_68">
         <property name="text">
          <string>5</string>
         </property>
        </widget>
       </item>
       <item row="0" column="7">
        <widget class="QLabel" name="label_69">
         <property name="text">
          <string>6</string>
         </property>
        </widget>
       </item>
       <item row="0" column="8">
        <widget class="QLabel" name="label_70">
         <property name="text">
          <string>7</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
